backend/src/llm/openai.py

```diff
@@ -28,16 +28,20 @@ async def chat(self, model, system_prompt: str, user_prompt: str, return_json=Fa
                     {"role": "user", "content": user_prompt},
                 ],
                 temperature=0,
-                response_format={"type": "json_object"} if return_json else NOT_GIVEN,
+                response_format={
+                    "type": "json_object"} if return_json else NOT_GIVEN,
             )
-            logger.info("OpenAI response: {0}".format(response))
             content = response.choices[0].message.content
+            logger.info(f"OpenAI response: Finish reason: {
```

Comment:
since this is here we don't need the log in on line 37 of `backend/src/router.py` because this is a duplicate of the best next steps response